mkdocs.yml

```diff
@@ -132,6 +132,9 @@ nav:
       - Community: Community.md
       - FAQ: FAQ.md
 
+hooks:
```

Comment:
great. we should use this for moving the `install.sh` script as well cc @ruben-arts :)

Reply:
Indeed: my first foray into working the `mkdocs` API. Not really worse than sphinx, but, ironically, much harder-to-read docs even though the API seems a little more sane/generalized.